params.py

```diff
@@ -74,9 +73,13 @@
 For the input of an experiment the 'current_set_up' list can be updated
 """
 SCORERS = {
-    'current_set_up': ['test'],
+    'current_set_up': ['openface', 'vggface', 'dummy'],
     'all': {
-        'test': TestModel()
+        'dummy': DummyModel(),
+        'openface': Deepface_Lib_Model(model=OpenFace.loadModel()),
```

Comment:
deze implementatie zorgt er trouwens wel voor dat al deze modellen geladen worden, ook as ze niet gebruikt worden. Onhandig als laden duur is?